### ISSUE TITLE
Add support for import custom launch scripts.

### ISSUE THREAD

**Issue by EndrII** [QuasarApp/CQtDeployer#486]
**Description of the problem**
Sometimes we need to run an application with a custom script. The current customScript option function can only add simple line into the script that's is not enough.

**Description of the solution**
Add support for import custom launch scripts for cqtdeployer.

**Example**
```
cqtdeployer -runScript myTargetMame;path/to/my/myCustomLaunchScript.sh 
```

**Comment by pzhlkj6612** [comment]
What about Powershell Core/PowerShell script (.ps1)?

**Comment by EndrII** [comment]
> What about Powershell Core/PowerShell script (.ps1)?

The PowerShel scripts is not universal therefore the bat format using as a executable format. If you want to add ps1 script to the distributive use the extraData and extraDataOut option. If you want run ps1 script automatically before starting main program you need use the runScript option.

## Examples:
### Add ps1 script into distribution.
```
cqtdeployer -extraData path/to/my/scripts.ps1 -extraDataOut "" 
``` 
### Add ps1 scripts into executable script

```
cqtdeployer -extraData path/to/my/scripts.ps1 -extraDataOut "" -runScript "myExecutable.exe;path/to/myBat.bat"
```

### myBat.bat
``` bat 
@echo off 
SET BASE_DIR=%~dp0

Powershell.exe -executionpolicy remotesigned -File  "%BASE_DIR%\scripts.ps1

call "%BASE_DIR%\cqtdeployer.exe" %*
```

**Comment by pzhlkj6612** [comment]
Got it, thank you.

**Comment by EndrII** [comment]
This feature is released for test in the cqtdeployer 1.5.0.15 alpha. 
See #487 for more information.
